api/src/pcapi/sandboxes/scripts/creators/industrial/create_user_offerers.py

```diff
@@ -0,0 +1,31 @@
+import logging
+
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers import models as offerers_models
+from pcapi.core.users import factories as users_factories
+
+
+logger = logging.getLogger(__name__)
+
+
+def create_user_offerers() -> None:
+    # Add a user and user_offerer for each offerer without one, except if the offerer has a provider
+    logger.info("create_user_offerers")
+    user = None
+    count = 0
+
+    query = (
+        offerers_models.Offerer.query.outerjoin(offerers_models.Offerer.UserOfferers)
+        .outerjoin(offerers_models.Offerer.offererProviders)
+        .filter(offerers_models.UserOfferer.id.is_(None), offerers_models.OffererProvider.id.is_(None))
+    )
+    for offerer in query:
+        if user is None:
+            user = users_factories.ProFactory(
+                email="user.offerer@example.com", firstName="Compte pro", lastName="Rataché"
```

Comment:
Je n'avais pas fait la faute :laughing: 
```suggestion
                email="user.offerer@example.com", firstName="Compte pro", lastName="Rattaché"
```

Reply:
ohnon la honte 🫣